plugins/optimization-detective/class-od-url-metrics-group.php

```diff
@@ -330,7 +330,7 @@ public function getIterator(): ArrayIterator {
 	/**
 	 * Counts the URL metrics in the group.
 	 *
-	 * @return int URL metric count.
+	 * @return int<0, max> URL metric count.
```

Comment:
There's also `positive-int` in PHPStan

Reply:
I think that is `int<1, max>`, isn't it?

Reply:
Right. `non-negative-int` is `0,max`